dlib/image_transforms/draw.h

```diff
@@ -225,6 +225,61 @@ namespace dlib
         }
     }
 
+// ----------------------------------------------------------------------------------------
+
+    template <
+        typename T, typename traits,
+        typename alloc
+    >
+    std::pair<long, long> compute_string_dimensions (
```

Comment:
It would be nicer to return a struct with explicitly named width and height fields.  You can still use structured bindings with a simple struct too

Reply:
Something like `string_dims` (following the convention of `chip_dims`?
Then, how about calling the function `compute_string_dims`?

Reply:
Works for me 😁

Reply:
Done :)